includes/admin/post-types.php

```diff
@@ -153,7 +153,10 @@ public function row_actions( $actions, $post ) {
 	 * @since 3.0.0
 	 */
 	public function bulk_actions() {
-
+		// Check user has permission to edit
+		if ( ! current_user_can( 'edit_posts' ) ) {
```

Comment:
Do we not need to check for `nonce` here?

Reply:
NO, this function call on page reload and take request from url. so there is no need.